MinecraftClient/ChatBot.cs

```diff
@@ -662,6 +664,45 @@ public bool GetInventoryEnabled()
         {
             return Handler.GetInventoryEnabled();
         }
+        public Dictionary<int, Container> GetInventories()
+        {
+            return Handler.GetInventories();
+        }
+        /// <summary>
+        /// Use item in hand
+        /// </summary>
+        protected void UseItemInHand()
+        {
+            Handler.UseItemOnHand();
```

Comment:
For consistency, could you also rename `UseItemOnHand` to `UseItemInHand`? Thanks 😉